fe/fe-core/src/main/java/com/starrocks/load/Load.java

```diff
@@ -619,8 +619,8 @@ private static Expr transformHadoopFunctionExpr(Table tbl, String columnName, Ex
                     if (funcExpr.hasChild(1)) {
                         exprs.add(funcExpr.getChild(1));
                     } else {
-                        if (column.getDefaultValue() != null) {
-                            exprs.add(new StringLiteral(column.getDefaultValue()));
+                        if (column.existBatchConstDefaultValue()) {
+                            exprs.add(new StringLiteral(column.getCalculatedDefaultValue()));
```

Comment:
This function is called many times. I think it's better to call this function once and assign its return to a temporary value.